tests/conftest.py

```diff
@@ -98,14 +146,29 @@ def loop(request):
 
 
 @pytest.fixture(scope='class')
-def setup_test_class_serverless(request, loop):
+def setup_test_class_serverless(request, loop, ssl_folder):
     request.cls.loop = loop
+    request.cls.ssl_folder = ssl_folder
 
 
 @pytest.fixture(scope='class')
-def setup_test_class(request, loop, kafka_server):
+def setup_test_class(request, loop, kafka_server, ssl_folder):
     request.cls.loop = loop
-    request.cls.kafka_host, request.cls.kafka_port = kafka_server
+    khost, kport, ksslport = kafka_server
+    request.cls.kafka_host = khost
+    request.cls.kafka_port = kport
+    request.cls.kafka_ssl_port = ksslport
+    request.cls.ssl_folder = ssl_folder
+
+    def _create_context(self):
```

Comment:
why so implicit?) you pass `ssl_folder` to test class. So can implement this helper method in `KafkaIntegrationTestCase` class.